### DISCUSSION TITLE
Lordaeron LK heroic10 berserk timer

### DISCUSSION THREAD

**Discussion by endershan** [Zidras/DBM-Warmane#170] [Q&A] [answered]
Hi, I recently confirmed that LK heroic10 on Lordaeron has a berserk timer of 15min, just same as heroic25. Could you change this in the next release? Thank you!

**Comment by Zidras** [comment]
Do you have a log/vid?
Indeed I have a comment with this question https://github.com/Zidras/DBM-Warmane/blob/ece4237ecce436e47e0bc5c81240b93d6221f812/DBM-Icecrown/TheFrozenThrone/LichKing.lua#L269

**Comment by endershan** [comment]
To be more accurate we only confirmed that it is longer than 14m43s, which is how long it took us to kill him. So my guess is they set the same berserk timer as heroic 25.
Here is the video link: https://www.bilibili.com/video/BV1iM411P75a/
The client is in Chinese but you can see the fight started at 01:19 and we beat him at 16:03.
If you have problem logging on the video site, I will try upload a version to Youtube.

**Comment by endershan** [comment]
P.S. Here is the warmane log: https://armory.warmane.com/pveladder/Lordaeron/details/4153887

**Comment by Zidras** [comment]
Thanks, I'll review everything until the end of the week

**Comment by Zidras** [comment, marked as answer]
Fixed on https://github.com/Zidras/DBM-Warmane/commit/0bc3ea65057bb7c9d52e7e6e1461c9a555fcc83d
